allennlp/modules/highway.py

```diff
@@ -22,8 +22,8 @@ class Highway(torch.nn.Module):
     Parameters
     ----------
     input_dim : ``int``
-        The dimensionality of :math:`x`.  We assume the input has shape ``(batch_size,
-        input_dim)``.
+        The dimensionality of :math:`x`.  We assume the input has shape ``(batch_size, *,
+        input_dim)``, where * is any number of dimensions.
```

Comment:
sphinx won't like the `*` here.  We've typically just use `...` to indicate some arbitrary number of dimensions.

Reply:
Sure, figured that would be changed, thanks!